.github/workflows/swagger.yml

```diff
@@ -0,0 +1,41 @@
+name: Open API Specification and Documentation
+on:
+  push:
+    branches:
+      - 'master'
+    paths:
+      - 'helpers/**'
+  pull_request:
+    branches:
+      - 'master'
+    paths:
+      - 'helpers/**'

```

Comment:
Remove this `paths` section to see if it can be executed without this.

Reply:
@ramrodo : Sir, can you pl review this. :slightly_smiling_face:

Reply:
Sure. I left a couple of comments more